openapi-v1.yaml

```diff
@@ -2677,6 +2679,36 @@ definitions:
         items:
           type: string
 
+  InvitationGroupShareEmail:
+    description: Encapsulates information regarding inviting people using email to share group, same permissions for all invitees
+    type: object
+    required:
+      - array_actions
+      - group_actions
+      - invitee_email_or_name
```

Comment:
Typo? the actual prop is `invitee_email_or_namespace ` not `invitee_email_or_name`.

Reply:
Namespace is an object according to my understanding "a space (an object) characterized by a name". This was understood by code: `// NamespaceName NamespaceName
// swagger:model NamespaceName
type Namespace struct {

	// unique id of namepace, user or organization
	UUID *uuid.UUID `json:"id,omitempty" gorm:"PRIMARY_KEY;size:36"`

	// user or organization name must be unique
	// Required: true
	// Max Length: 20
	// Min Length: 4
	// Pattern: ^[\w.\-]+$
	NamespaceName string `json:"namespace_name" gorm:"unique_index"`

	User     User `json:"-" gorm:"association_autocreate:false;association_autoupdate:false;foreignkey:UserUUID"`
	UserUUID *uuid.UUID

	Organization     Organization `json:"-" gorm:"association_autocreate:false;association_autoupdate:false;foreignkey:OrganizationUUID"`
	OrganizationUUID *uuid.UUID

	CreatedAt time.Time  `json:"-"`
	UpdatedAt time.Time  `json:"-"`
	DeletedAt *time.Time `json:"-"`
}`

So I though that a user or an organization is a namespace. 
The invitee_email_or_name field should there be like invitee_email_or_namespacename (namespace exists in namespace struct above) so I chose name as a shorter word for namespace name. What do you think @alekshs ?

Reply:
Both names make sense. We just need to make things consistent. They aren't consistent at the moment and as a result client generators throw an error.

Reply:
ok changing this to namespace

Reply:
Sorry @alekshs did not see that invitee_email_or_namespace was used below. I am fixing this.